tests/ice40/mul_pow.ys

```diff
@@ -0,0 +1,7 @@
+synth_ice40
+#equiv_opt -assert -map +/ice40/cells_sim.v synth_ice40
```

Comment:
... and run `synth_ice40` with the `-dsp` option to infer multipliers.

Reply:
for now the result is the same. Will change this case.

Reply:
That's because the multiply operation isn't wide enough:
```
Change circuit to be wider:

module top
(
 input [5:0] x,
 input [5:0] y,

 output [11:0] A,
 );

assign A =  x * y;

endmodule
```
now works.